app/models/season/phase_switcher.rb

```diff
@@ -4,6 +4,17 @@ def self.season
     Season.current
   end
 
+  def self.destined(phase)
+    unless phase.to_sym.in?(whitelisted_phases)
+      raise ArgumentError
+    else
+      self.public_send(phase)
+    end
+  end
+
```

Comment:
- Can it be that the raising Error line slows down the switching? It looks like it took a whole 4ms more to switch after I added the `raise`.  
- Also, not sure if this is the right Error to raise.
- Before that I had a silent fail - not sending,  and not telling so.
I liked that, because a) in the controller we prevent access to the method in production env and b) if someone tries to do something malicious, a smoke screen serves them right LOL